.github/workflows/build-and-test.yaml

```
@@ -0,0 +1,37 @@
+name: build-and-test
+
+on:
+  push:
+    branches:
+      - main
+      - tier4/proposal
+  schedule:
+    - cron: 0 19 * * * # run at 4 AM JST
+  workflow_dispatch:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ros:galactic
+    steps:
+      - name: Checkout repository
```

https://github.com/autowarefoundation/autoware.universe/pull/307

```suggestion
      - name: Check out repository
```